src/components/options-list/index.tsx

```diff
@@ -71,6 +79,18 @@ export const OptionsList = ({
 }: OptionsListProps) => {
     const ref = useRef<FlatList>(null);
```

Comment:
Where are you using this ref?

Reply:
look at the FlatList and renderItem, there is the `ref`